packages/common/test/pages/_internal/PageUiSchemaEdit.test.ts

```diff
@@ -245,6 +245,7 @@ describe("buildUiSchema: page edit", () => {
                 control: "hub-field-input-location-picker",
                 extent: [],
                 options: [],
+                noticeTitleElementAriaLevel: 3,
```

Comment:
See my comment in the [opendata-ui PR](https://github.com/ArcGIS/opendata-ui/pull/14359#discussion_r1893210940)